css/index.css

```diff
@@ -1,3 +1,489 @@
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+    margin: 0;
+    padding: 0;
+    border: 0;
+    font-size: 100%;
+    font: inherit;
+    vertical-align: baseline;
+}
 
+/* HTML5 display-role reset for older browsers */
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+    display: block;
+}
+
+body {
+    line-height: 1;
+}
+
+ol,
+ul {
+    list-style: none;
+}
+
+blockquote,
+q {
+    quotes: none;
+}
+
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+    content: '';
+    content: none;
+}
+
+table {
+    border-collapse: collapse;
+    border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html,
+body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+.logo {
+    display: flex;
+
+}
+
+.container {
+    max-width: 1200px;
+    width: 75%;
+    margin: 0 auto;
+}
+
+nav {
+    display: flex;
+    justify-content: space-between;
+}
+
+nav a {
+    text-decoration: none;
+    color: lightgray;
+    padding-right: 60px;
+}
+
+.demand-main {
+    display: flex;
+    justify-content: space-between;
+}
+
+.demand {
+    display: flex;
+    flex-direction: column;
+    font-weight: bold;
+    text-align: center;
+    align-items: center;
+    margin-bottom: 50px;
+}
+
+h1 {
+    font-size: 90px;
+    display: block;
+}
+
+.demand button {
+    display: block;
+    font-size: 20px;
+    background-color: white;
+    padding: 5px;
+    border-radius: 5%;
+    width: 150px;
+
+
+}
+
+button {
+    border: 1px solid black;
+}
+
+.features-about {
+    display: flex;
+    justify-content: space-around;
+    margin: 30px 0px;
+    line-height: 20px;
+}
+
+.services-vision {
+    display: flex;
+    justify-content: space-around;
+    margin: 30px 0px;
+    line-height: 20px;
+}
+
+.services {
+    margin: 0px 20px 0px 0px;
+}
+
+.product {
+    margin: 0px 20px 0px 0px;
+}
+
+.features {
+    display: flex;
+    flex-direction: column;
+    margin-right: 20px;
+
+}
+
+.about {
+    display: flex;
+    flex-direction: column;
+
+}
+
+.main-header header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.main-header {
+    padding: 30px 0;
+
+}
+
+.section {
+    display: flex;
+    justify-content: space-between;
+}
+
+.sectiion2 {
```

Comment:
Beware of spelling errors!

By the way, we need to figure out how to set up linting in Brackets.